src/streamlink/plugins/teamliquid.py

```diff
@@ -1,4 +1,5 @@
 """
+$description Live streams from Team Liquid, a multi-regional professional e-sports organisation based in the Netherlands.
```

Comment:
> Live streams from Team Liquid

it is not just them, there are more https://tl.net/video/streams/

but this plugin should be removed

Reply:
Let's leave it then - I'll submit a PR for removal of this plugin.